src/components/rockets/List.jsx

```diff
@@ -0,0 +1,42 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import Rocket from './Rocket';
+import { getRockets, rocketAPICall } from '../../redux/rockets/rockets';
+
+const RocketList = () => {
+  const dispatch = useDispatch();
+  const rocketStorage = useSelector((state) => state.rocketReducer);
+
+  useEffect(() => {
+    async function getRocketData() {
+      if (rocketStorage.length === 0) {
```

Comment:
- [ ] Nice condition before performing API call. But what happens if our list of rockets is not empty and there are some updates done on the server. In this case, we will miss out on the updates since our list is not empty.